features/step_definitions/comment_steps.rb

```diff
@@ -45,7 +45,7 @@
 
 When /^I set up the comment "([^"]*)" on the work "([^"]*)"$/ do |comment_text, work|
   work = Work.find_by_title!(work)
-  visit work_url(work)
+  visit work_path(work)
```

Comment:
In a JS-enabled test, the domain is set to `example.com` and the absolute `work_url` will include that. I saw #2594 where we got around this with `visit work_url(work).gsub("http://www.example.com","")`, but I think `work_path` is nicer.